grunttasks/copyright.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.config('copyright', {
+    app: {
+      src: [
+        "{,bin/,config/,db/,scripts/}*.js"
+      ],
+      options: {
+        pattern: "This Source Code Form is subject to the terms of the Mozilla Public"
+      }
+    },
+    tests: {
+      src: [
+        'test/{local,backend}/*.js'
```

Comment:
What's interesting about this, is that it doesnt seem to check the test/*.js files (only the sub directories).
Not sure how easily fixable that is, since it looks like test/client-then.js, and test/fake.js have a different license (public domain) than the test/ptaptest.js (mpl v2).

Possibly a bit ugly to do something like:
```js
src: [
  'test/{local,backend}/*.js',
  'test/{client-then,fake}.js'
]
```

... and then change the `app.src` path above to:
```js
src: [
  '{,scripts/}*.js',
  'test/ptaptest.js'
]
```